runtime/gc_realtime/ConfigurationRealtime.cpp

```diff
@@ -108,7 +108,9 @@ MM_Heap *
 MM_ConfigurationRealtime::createHeapWithManager(MM_EnvironmentBase *env, uintptr_t heapBytesRequested, MM_HeapRegionManager *regionManager)
 {
 	MM_GCExtensionsBase *extensions = env->getExtensions();
-
+	J9JavaVM *vm = (J9JavaVM *)extensions->getOmrVM()->_language_vm;
+	/* set indexableObjectLayout = J9IndexableObjectLayout_NoDataAddr_Arraylet (metronome GC)*/
```

Comment:
the code line is fairly self explanatory, but if you want to add a comment let's try be a bit more creative rather than just repeating the code:

"Let VM know that Metronome has discontiguous indexable object (arraylet layout)"